app/services/albums.js

```diff
@@ -1,9 +1,61 @@
 const request = require('request-promise'),
   logger = require('../logger'),
-  errors = require('../errors');
+  errors = require('../errors'),
+  config = require('../../config'),
+  error = require('../errors'),
+  urlApi = config.common.apiAlbums.url,
+  { Album } = require('../models'),
+  servicesUser = require('../services/users');
 
-exports.getAlbums = url =>
-  request(url).catch(error => {
-    logger.error(error);
-    throw errors.albumsApiError(error.message);
+const createAlbums = albumData =>
+  Album.create(albumData)
+    .then(() => `the album '${albumData.name}' was purchased correctly`)
+    .catch(err => {
+      throw error.databaseError(err.message);
+    });
+
+const getAlbum = url => {
+  const options = {
+    uri: url,
+    json: true
+  };
+  return request(options).catch(err => {
+    logger.error(err);
+    throw errors.albumsApiError(err.message);
   });
+};
+
+const albumPurchased = (albumId, userId) =>
+  Album.findAndCountAll({
+    where: { albumId, userId },
+    attributes: ['id']
+  })
+    .then(result => result.count !== 0)
+    .catch(err => {
+      throw error.databaseError(err.message);
+    });
+
+exports.getAlbums = url => getAlbum(url);
```

Comment:
I wrote you something for the private, when you fix it let me know

Reply:
perfect!